tests/main/appstream-id/task.yaml

```diff
@@ -1,8 +1,6 @@
 summary: Verify AppStream ID integration
-# ubuntu-*-32: the snap used in the test was published only for amd64
 # fedora-*: uses GNU netcat by default
```

Comment:
While at it, can you update the comment that Fedora uses nmap's netcat instead of the GNU one?